db/migrate/20230922102926_revoking_not_used_authorization_request_with_no_datapass.rb

```diff
@@ -0,0 +1,21 @@
+class RevokingNotUsedAuthorizationRequestWithNoDatapass < ActiveRecord::Migration[7.0]
+  def change
+
+    used_ar_id_with_no_datapass = %w[
+      14003128-5eed-4087-b04c-b4c6e5962040
+      61da2c91-1c49-4ac2-8012-efabac411b44
+      9991734e-317b-4110-8a38-aad86edce633
+      aec4345f-155a-47c7-9bc8-6acd671e945d
+      f7fe5da2-48cc-4675-8fad-25cba4ef0f33
+    ]
+
+    not_used_ars_with_no_datapass = AuthorizationRequest
+      .where(external_id: nil)
+      .where.not(id: used_ar_id_with_no_datapass)
+
+    not_used_ars_with_no_datapass.each do |authorization_request|
```

Comment:
```suggestion
    not_used_ars_with_no_datapass.find_each do |authorization_request|
```

plus conventionnel pour faire du batch